### ISSUE TITLE
Initial speed in excess of 15,000 kps in this saved game

### ISSUE THREAD

**Issue by evertvorster** [vegastrike/Vega-Strike-Engine-Source#493]
[Evert_Mission2.zip](https://github.com/vegastrike/Vega-Strike-Engine-Source/files/6787213/Evert_Mission2.zip)

The game is saved on a refinery... mechanist, I think. When I launch from this installation, I have an initial ship speed in excess of 15,000kps. It takes a while to bleed off all that speed, but then the game handles normally. 

I have flown some distance towards Sol, and landed on a planet. Taking off from a planet sets the kps to 0, which is as it should be. 

The save game is fairly old, I have been using it to explore the universe with Python 3 from last year, and made over 11,000,000 credits the hard way.

**Comment by stephengtuggy** [comment]
Yes, one of my saved games dumps me out in space going over 20,000 kps. I think the problem is that your initial speed after undocking is set relative to an arbitrary object in the system, which could be (and sometimes is) another ship. If that other ship is going 20,000 kps absolute speed, now so are you.

**Comment by evertvorster** [comment]
Interesting...

Can't it pick the closest object? It's almost guaranteed to be a base or
planet...

Thanks for the explanation.
Evert Vorster
Awesome Chapters Tours
http://www.awesomechapters.com
Tel: +264 (0) 811477690

**Comment by BenjamenMeyer** [comment]
we should perhaps have it always pick the launch point.

**Comment by stephengtuggy** [comment]
> we should perhaps have it always pick the launch point.

Yes. That is exactly what it should do. See also #185

**Comment by ministerofinformation** [comment]
On a purely informational note: The velocity display is in m/s, not km/s.

Re: picking the launch point - this is actually vitally important when the launch point is moving - so we should definitely figure out how the selection is failing or if the save game format/read is encoding/retrieving incorrect/insufficient data

**Comment by evertvorster** [comment]
This is why I added in the savegame to the bug report, hopefully someone
can reproduce the error?
It's pretty solid on my system with that savegame.


Evert Vorster
Awesome Chapters Tours
http://www.awesomechapters.com
Tel: +264 (0) 811477690

**Comment by stephengtuggy** [comment]
@evertvorster Yes, I am able to reproduce it on my system with one of my saved games.

**Comment by royfalk** [comment]
This bug is still around. Encountered recently with my saved games.
